docs/src/technologies/device_tree_overlays_pt.md

```diff
@@ -0,0 +1,151 @@
+<!--
+    Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Device tree overlays for passthrough
+
+The device tree blob (.dtb file) is a data structure that describes the hardware 
+components of a particular system so that the operating system can use and manage 
+those components. For passthrough, the hardware description in the host needs
+some modifications, which include:
+
+- Removing the driver of the device to passthrough by assigning a dummy string
+  to the *compatible* property.
+- Adding the *iommus* property to the device to passthrough.
+- Removing or adding other properties that cause conflicts during the passthrough.
+
+Modifying the host device tree could be done by applying patches
+to the .dts (Device Tree Source) files. Nevertheless, this option is not 
+scalable if we need to apply different patches to the same .dts from 
+different configurations.
+
+A better and more scalable approach to modifying a device tree is using device
+tree overlays. The device tree overlay contains information about the nodes
+to modify (in nodes called *fragment@0 ...*) and the overlay of the 
+properties that we want to affect. More information about the overlays
```

Comment:
For more information on the overlays, see [Overlay notes](https://www.kernel.org/doc/Documentation/devicetree/overlay-notes.txt).